src/odinapi/templates/dataaccess.html

```diff
@@ -357,6 +364,12 @@ <h2>Downloadable examples</h2>
 <a href="{{ url_for('static', filename='examples/get_l1b_for_period.py') }}">
   here</a>.
 </p>
+<p>
+The <em>Verification Data Set</em> API provides functionality for comparing
+Odin/SMR data with data from other instruments. An example Python program
+illustrating its use can be dowloaded
+<a href="{{ url_for('static', filename='examples/vds_example.py') }}">
+  here</a>.
```

Comment:
```suggestion
<a href="{{ url_for('static', filename='examples/vds_example.py') }}">here</a>.
```

linebreak of html at 80 doesn't make any sense. also multiple spaces but perhaps there's a need for an `&npsp;` between the link and "downloaded"?